Resources/Prototypes/_Goobstation/Entities/Structures/Wallmounts/Signs/posters.yml

```
@@ -0,0 +1,89 @@
+    sprite: _Goobstation/Structures/Wallmounts/posters.rsi
+    state: poster64_contraband
+    
+- type: entity
+  parent: PosterBase
+  id: PosterContrabandYakub
+  name: "Yakub Poster"
+  description: "Your yakubian powers of tricknology imbued into you by Yakub strengthen as you stare into the poster"
+  components:
+  - type: Sprite
+    sprite: _Goobstation/Structures/Wallmounts/posters.rsi
+    state: poster65_contraband
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)